### DISCUSSION TITLE
PCB and housing

### DISCUSSION THREAD

**Discussion by fscii** [Montecri/GNSSTimeServer#17] [General]
First of all the project looks great, I can't wait to do this one.  Now onto the questions.

1.  Does the PCB made for the wooden housing also have the same fit into the 3d printed case?
2.  How do I get the PCB?  I've ordered custom PCB;s from those who had them on hand but I've never had to take Gerber files and try to get a single PCB made :/  Do you have any for sale?  If not which website/supplier should I go to for the best price and turnaround for a single board or two?

Thanks!

**Comment by Montecri** [comment]
Hi!

Let me try to answer:

1) No, it's a different, custom version of the time server.

2) Unfortunately, no PCB for sale or Gerber files at this moment. I'm working on that for the second half of this year. Today what I use is a generic 9 cm x 7 cm prototyping board with a ESP32 specific footprint.

This one here specifically:

![image](https://github.com/Montecri/GNSSTimeServer/assets/38574378/43d19d45-2361-4808-93c1-75f81b485ce5)

Which I purchased here:

https://www.aliexpress.com/item/32838846336.html?spm=a2g0o.order_list.order_list_main.5.1ea21802uo1A4I

Used a heat gun and solder paste to solder the ESP32 module and the other components by hand with a soldering iron.

Regards,

- Cristiano
